src/components/search-box/SearchBoxInput.vue

```diff
@@ -128,5 +183,20 @@ defineExpose({ focus })
     <div v-if="canClose" class="lupa-close-search-container" @click="$emit('close')">
       <span v-if="labels.close" class="lupa-close-label">{{ labels.close }}</span>
     </div>
+    <div v-if="isVoiceSearchEnabled">
+      <button 
+        @click="openVoiceSearchDialog" 
+        class="voice-search-button"
```

Comment:
Plugin prefixes all of its classes with `lupa` to avoid breaking the styles. It should be `lupa-voice-search-button`